wordpress-mobile/WordPress-Android
# Wp support forum: add tracks

This PR adds tracks to Support Forum card on Help screen

Fixes #17809 

<img width=320 src=https://user-images.githubusercontent.com/990349/214458783-e751b04a-502d-42d9-9a44-b9995a7b8eee.png />

To test:

- Navigate to Me > App Settings > Debug settings
- Enable wordpress_support_forum_remote_field under Remote features and Restart the app (scroll down if required)
- Navigate to Me > Help & Support
- Verify that the screen is as shown above
- Tap on the Community forums row
- Verify that it opens the new Mobile Forum on Wordpress.org in the browser
- Verify the following tracks is fired in the Logcat in Android Studio
🔵 Tracked: support_open_mobile_forum_tapped
- Alternatively, Turn on “Collect information” in Account > App Settings > Privacy settings
- Navigate to Me > Help and Support > Application log 
- Find 🔵 Tracked: support_open_mobile_forum_tapped

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
